src/predtimechart.js

```diff
@@ -1021,20 +798,6 @@ const App = {
             }
             Promise.all(promises).then((values) => {
                 console.debug(`fetchDataUpdatePlot(${isFetchFirst}, ${isFetchCurrentTruth}): Promise.all() done. updating plot`, values);
-
-                // update user ensemble model if any
-                if (this.isUemEnabled && this.state.models.includes(USER_ENSEMBLE_MODEL.name)) {
-                    try {
-                        this.state.forecasts[USER_ENSEMBLE_MODEL.name] = _calcUemForecasts(USER_ENSEMBLE_MODEL.models, this.state.forecasts);  // replaces if present
-                        USER_ENSEMBLE_MODEL.lastError = null;
-                        console.debug('fetchDataUpdatePlot(): forecasts:', this.state.forecasts[USER_ENSEMBLE_MODEL.name]);
-                    } catch (error) {
-                        USER_ENSEMBLE_MODEL.lastError = error;
-                        console.error(`fetchDataUpdatePlot(): error calling _calcUemForecasts(): ${error}`);
-                    }
-                }
-
-                this.updateModelsList();
```

Comment:
double checking on the deletion of `this.updateModelsList();`.  Seems like we might still want that?  I'm not confident.